lib/Runtime/Base/FunctionBody.h

```diff
@@ -2374,7 +2424,7 @@ namespace Js
         void SetCheckCodeGenEntryPoint(FunctionEntryPointInfo* entryPointInfo, JavascriptMethod entryPoint);
 
 #if ENABLE_NATIVE_CODEGEN
-        typedef void (*SetNativeEntryPointFuncType)(FunctionEntryPointInfo* entryPointInfo, Js::FunctionBody * functionBody, Js::JavascriptMethod entryPoint);
+        typedef void(*SetNativeEntryPointFuncType)(FunctionEntryPointInfo* entryPointInfo, Js::FunctionBody * functionBody, Js::JavascriptMethod entryPoint);
```

Comment:
nit: Did VS make this change? I think the space after the return type reads better.